scripts/generateApiClientModules.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+wget http://central.maven.org/maven2/io/swagger/swagger-codegen-cli/2.3.1/swagger-codegen-cli-2.3.1.jar -O swagger-codegen-cli.jar
+java -jar swagger-codegen-cli.jar generate -i http://genomenexus.org/v2/api-docs -l java -c genomeNexusPublicApiClientConfig.json -o ../genomeNexusPublicApiClient
```

Comment:
let's change this to take some parameter e.g. `$1`, could default to https://genomenexus.org/v2/api-docs

Reply:
also it assumes this run from the directory this script is residing in, so prolly good to cd to the `SCRIPT_DIR` first, e.g.: https://github.com/cBioPortal/cbioportal-frontend/blob/master/scripts/generate_spec_files.sh#L2